### PR TITLE
fix(plugin): enable error-mapper

### PR DESCRIPTION
**Pull request by Birkbjo**

### CODE REVIEW

File: test/routes/v2/organisations.js [hunk 1]
```diff
@@ -285,8 +283,8 @@ describe('v2/organisations', () => {
                 },
             }
             const res = await server.inject(opts)
-            expect(res.statusCode).to.equal(409)
-            expect(res.result.message).to.be.equal('Organisation not found')
+            expect(res.statusCode).to.equal(404)
+            expect(res.result.message).to.be.equal('Not Found')
```
Comment (erikarenhill) [inline test/routes/v2/organisations.js]: being picky about the commit message, the fixed test should not belong to "fix(plugin): enable error-mapper" ? But instead go in an own commit.

Comment (Birkbjo) [reply test/routes/v2/organisations.js]: I will deduplicate them, but it's related as this would return a 500 error without the error-mapper.
